external/radiation/radiation/radiation_astronomy.py

```diff
@@ -268,8 +266,6 @@ def sol_update(self, jdate, kyear, deltsw, deltim, lsol_chg, rank, solar_data):
 
                                 else:
                                     i -= 1
-        else:
```

Comment:
This is not in Fortran and I'm not sure why it was added to the port. It had the unintended side effect of overriding the solar constant with a fixed value for any radiation calculation after the first one.